### DISCUSSION TITLE
「Pwn」粗浅分析 House Of Muney 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#20] [Announcements]
# 「Pwn」粗浅分析 House Of Muney 🐱 ネコのメモ帳

前几天 ZBR 发了这个 repo，我寻思没听过，看着攻击能力还挺强的，于是浅浅分析一下。

https://nova.gal/blog/2024/09/12/%E3%80%8CPwn%E3%80%8D%E7%B2%97%E6%B5%85%E5%88%86%E6%9E%90-House-Of-Muney

**Comment by huhusmang** [comment]
同学保到哪里了🐱，同是 vul337 实习过的

**Reply by huhusmang** [reply]
follow 已订阅~

**Reply by MuelNova** [reply]
pku cs

**Reply by huhusmang** [reply]
恭喜恭喜！
